backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/X_AD_Process.java

```diff
@@ -632,4 +632,34 @@ public boolean isLogWarning()
 		return get_ValueAsBoolean(COLUMNNAME_IsLogWarning);
 	}
 
+	@Override
+	public void setStoreProcessResultFileOn (final @Nullable java.lang.String StoreProcessResultFileOn)
+	{
+		set_Value (COLUMNNAME_StoreProcessResultFileOn, StoreProcessResultFileOn);
+	}
+
+	@Override
+	public java.lang.String getStoreProcessResultFileOn()
+	{
+		return get_ValueAsString(COLUMNNAME_StoreProcessResultFileOn);
+	}
+
+	// FIXME make sure they get generated!
+	public static final String STOREPROCESSRESULTFILEON_Server = "S";
```

Comment:
pls make sure you generate the X_AD_Process class

Reply:
done